lib/presentation/resources/todo_cards/todo_crads_cubit.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:async';
+
+import 'package:clean_architecture_demo/domain/repository/todo_repository.dart';
+import 'package:clean_architecture_demo/presentation/resources/todo_cards/todo_cards_state.dart';
+import 'package:clean_architecture_demo/presentation/utils/ui_utils.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class TodoCardsCubit extends Cubit<TodoCardsState> {
+  TodoCardsCubit({required this.todoRepository})
+      : super(const GetTodoCardsInProgress());
+
+  final TodoRepository todoRepository;
+
+  void getTodoCards() async {
+    emit(const GetTodoCardsInProgress());
+    final result = await todoRepository.getTodoList();
+
+    if (result.right != null) {
+      emit(GetTodoCardsSuccess(todoCards: result.right));
+    } else {
+      showLongToast('Failed to fetch Todo list  : ${result.left.toString()}');
```

Comment:
@dilumdesilva can you pick this from localisation file after setting up localisation.

Reply:
Anyway, here we don't have context so localisation cannot be invoked. I will just show the error